backend/lib/server_code/api/socket.js

```diff
@@ -0,0 +1,23 @@
+const socketIo = require("socket.io");
+const http = require("http");
+const { app } = require("../../../api_server");
+
+const SOCKET_PORT = process.env.PORT || 4001;
+
+const server = http.createServer(app);
+const socket = socketIo(server);
+
+socket.on("connection", socket => {
```

Comment:
lets talk about it

Reply:
done... Socket is created on newgame or joining game